otter/convergence/transforming.py

```diff
@@ -66,6 +68,28 @@ def optimize_steps(clb_steps):
 _register_bulk_clb_optimizer(RemoveNodesFromCLB, 'node_ids')
 
 
+def _register_bulk_rcv3_optimizer(step_class):
+    """
+    Merge together multiple RCv3 bulk steps.
+
+    :param step_class: One of :obj:`BulkAddToRCv3` or :obj:`BulkRemoveFromRCv3`
```

Comment:
A specific type signature would be helpful here. `:param type step_class:` would highlight the fact that you literally want a `type` object.